src/helpers/messageHelper.js

```diff
@@ -0,0 +1,64 @@
+const constants = require('./constants.js');
+
+const messageIsFromABot = function (event) {
+  if (event.type === 'message' && (event.subtype === 'bot_message' ||
+      (event.bot_profile !== null && event.bot_profile !== undefined && event.bot_profile.name == 'bot'))) {
+    return true;
+  }
+  return false;
+}
+
+const getMessageResponse = function (event) {
+  // console.debug(JSON.stringify(event, null, 2))
+  // console.debug(process.env.ROBOT_NAME);
+  const message = event.text;
+
+  const whereIs = new RegExp(`where is ${process.env.ROBOT_NAME}`, 'i')
+  const wheres = new RegExp(`where's ${process.env.ROBOT_NAME}`, 'i')
```

Comment:
To limit the scope of the next PR, since I've already got some things working with the ENV variable, would you be content to have an Issue created to get the bot name from Slack and have that change made in a PR separate from a PR that will address some/all of #38?

Reply:
Sure that works for me.